docs/index.md

```diff
@@ -28,47 +28,47 @@ A line of text explaining snippet below…
 >>> from dataclasses import dataclass
```

Comment:
Correct placeholder text for library description, pros features and code example.